### PR TITLE
Training material for Galaxy and Pulsar on Surf Research Cloud

### PR DESCRIPTION
**Pull request by mirelaminkova**
Tutorial on how to start a Galaxy and Pulsar workspace on Surf Research Cloud

Training material for researchers using SURF Research Cloud who need to know how to start workspaces. 
- [x] @hexylena to add information about the Pulsar catalog item

### CODE REVIEW

**Comment by hexylena** [comment]
@shiltemann if you have time for reviewing this maybe?

**Review by hexylena — APPROVED** [review]
fantastic to see  🇳🇱

File: topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+> 
+> 3. You will need to `sudo su` to do anything useful. Do that now.
+> 4. `cd /srv/galaxy/` to move into the directory Galaxy configuration is stored.
+> 5. The configuration is discussed fully in the Pulsar information, but it will be briefly covered here as well. Generally there are a few steps that must be followed:
+>
+>    - A runner must be registered
+>    - A destination/environment must be added with the pulsar details
+>    - Some tools should be redirected to this Pulsar
+>
+>    Here is an example of what those changes *might* look like in your Galaxy node. ([FAQ: how to read a diff]({% link topics/admin/faqs/diffs.md %})). In this example our pulsar node was called `p20` but that will be different for you.
```
Comment (shiltemann) [inline topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.md:151]: FAQs can be included as tip boxes easily as well:

`{% snippet topics/admin/faqs/diffs.md %}`
```suggestion
>    Here is an example of what those changes *might* look like in your Galaxy node.  In this example our pulsar node was called `p20` but that will be different for you.
> 
>    {% snippet topics/admin/faqs/diffs.md %} 
>
```

File: topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+  GDPR: General Data Protection Regulations
+  SRAM: SURF Research Access Management
+  CO: Collaborative Organisation
+
+follow_up_training:
+  - type: "internal"
+    topic_name: admin
+    tutorials:
+      - surf-research-cloud-galaxy
+---
```
Comment (shiltemann) [inline topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.md:44]: Should these tutorials be listed in the "Running Galaxy on the Cloud" section of the GTN admin topic page? 

If so, please add `subtopic: cloud` to the metadata (now they are placed at bottom in the 'other' section)

File: topics/admin/tutorials/surf-research-cloud-galaxy/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  GDPR: General Data Protection Regulations
+  SRAM: SURF Research Access Management
+  CO: Collaborative Organisation
+
+follow_up_training:
+  - type: "internal"
+    topic_name: admin
+    tutorials:
+      - surf-research-cloud-pulsar
+---
```
Comment (shiltemann) [inline topics/admin/tutorials/surf-research-cloud-galaxy/tutorial.md:46]: add some tags to metadata? maybe 'deploying' as I see on others? anything else?

```
tags:
 - deploying
```

File: events/2025-01-28-src.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
+---
+layout: event
+
+# Status of this page, remove these lines when ready
+draft: true  # will hide your event from the GTN events list
```
Comment (shiltemann) [inline events/2025-01-28-src.md:5]: do you want to keep it as draft? feel free to remove or keep as you see fit

File: events/2025-01-28-src.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+        
+      - name: surf-research-cloud-pulsar
+        topic: admin
+        time: "14:10 - 14:40"
+        
+      - type: custom
+        name: Coffee Break
+        time: "14:40"
+      - name: mapping
+        type: custom
```
Comment (shiltemann) [inline events/2025-01-28-src.md:89]:
```suggestion
        topic: sequence-analysis
```
(to link to the tutorial proper)

File: events/2025-01-28-src.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        time: "14:40 - 15:40"
+
+      - name: an-introduction-to-genome-assembly
+        type: custom
```
Comment (shiltemann) [inline events/2025-01-28-src.md:93]:
```suggestion
      - name: general-introduction
        topic: assembly
```
(to link to the tutorial proper)

File: events/2025-01-28-src.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+# Status of this page, remove these lines when ready
+draft: true  # will hide your event from the GTN events list
+status: wip  # add a banner that warns that the contents of the page are still subject to change
+
+# Description of your event
+title: Galaxy at SURF Research Cloud workshop
+description: |
+  Half-day workshop demonstrating the local Galaxy instance built for users of SURF Research Cloud. 
+
+#cover-image:         # image for your corse, put in 'events/images' folder
```
Comment (shiltemann) [inline events/2025-01-28-src.md:13]: would you like to use the SURF logo image you included for the tutorial on the event page, or happy with the default GTN image?

File: topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.md (hunk 1 above)
Comment (hexylena) [reply topics/admin/tutorials/surf-research-cloud-pulsar/tutorial.md:151]: yeah this one was my fault. thanks!

**Review by mirelaminkova — COMMENTED** [review]
Addition of tags, subtopics, and key points according to @shiltemann 's feedback

**Review by hexylena — APPROVED** [review]
Looks good from my side :) @shiltemann can merge if she's happy

**Review by shiltemann — APPROVED** [review]

**Comment by shiltemann** [comment]
:tada: Thanks a lot @mirelaminkova and @hexylena !

**Pull request merged**